// tratando erros 
// try: tentar

try {
    console.log(naoEx);
} catch(e) { // (e) ou (err) esse é o bloco a ser executado caso aconteça o erro 
    console.log('naoEx não existe');

}